build.gradle

```diff
@@ -46,8 +46,8 @@ allprojects {
         resolutionStrategy {
             // Force 2.11.4 until upgrade is done
             // Jackson is to be locked due to conversion issue with Double<>BigDecimal
-            force "com.fasterxml.jackson.core:jackson-databind:2.11.4"
-            force "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.11.4"
+            force "com.fasterxml.jackson.core:jackson-databind:2.15.4"
```

Comment:
It would be better to remove the pin here. But the dependency of conductor might block it. There is an ongoing effort to remove the conductor (#5).

Reply:
Yes. Removing the pin results in the same error as described in the description for following tests
![image](https://github.com/user-attachments/assets/5f42a6e8-9dce-4c1e-acf5-804a728bef88)